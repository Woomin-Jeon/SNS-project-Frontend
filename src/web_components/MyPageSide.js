import React from 'react';

function MyPageSide({
  currentUserState,
}) {
  return (
    <div className="mypage-side-banner">
      <div>
        <i className="fas fa-globe-americas"/>
        <span className="mypage-side-banner-introduce">소개</span>
        <div className="mypage-side-banner-utter">
          간단한 소개를 추가하여 회원님에 대해 자세히 알려주세요
        </div>
        <div className="mypage-user-information">
          <i className="fas fa-birthday-cake"/>
          <div>{currentUserState.birth}</div> <br />
          <i className="fas fa-map-marker-alt"/>
          <div>{currentUserState.location}</div> <br />
          <i className="far fa-envelope"/>
          <div>{currentUserState.email}</div>
        </div>
      </div>
    </div>
  );
}

export default MyPageSide;
