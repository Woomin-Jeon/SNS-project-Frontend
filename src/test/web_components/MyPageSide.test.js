import MyPageSide from "../../web_components/MyPageSide";
import React from "react";
import {shallow} from 'enzyme';

describe('MyPageSide', () => {
  let currentUserState;

  beforeEach(() => {
    currentUserState = {
      birth: '1996-04-21',
      location: '대한민국',
      email: 'dal96k@hanmail.net',
    }
  });

  it('renders well', () => {
    const component = shallow(
      <MyPageSide
        currentUserState={currentUserState}
      />
    );

    expect(component).toMatchSnapshot();
  });
});
