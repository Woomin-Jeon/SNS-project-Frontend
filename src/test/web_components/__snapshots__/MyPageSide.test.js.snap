// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyPageSide renders well 1`] = `ShallowWrapper {}`;
